### PR TITLE
sysext: post-merge fixups

### PR DESCRIPTION
**Pull request by poettering**
Primarily addressing @yuwata's comment:

https://github.com/systemd/systemd/pull/18181#pullrequestreview-570826113

But also adds a `--force` switch for ignoring license incompatibilities

### CODE REVIEW

File: man/systemd-sysext.xml [hunk 1]
```diff
@@ -218,6 +219,14 @@
         output style, or explicitly disabling JSON output.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--force</option></term>
+
+        <listitem><para>When merging system extensions into <filename>/usr/</filename> and
+        <filename>/opt/</filename>, ignore version incompatibilities, i.e. force merging regardless whether
```
Comment (bluca) [inline man/systemd-sysext.xml]: nit: s/regardless whether/regardless of whether the/

**Comment by bluca** [comment]
Compiler's unhappy:

```
../src/sysext/sysext.c:1029:43: error: unused variable 'images' [-Werror,-Wunused-variable]
        _cleanup_(hashmap_freep) Hashmap *images = NULL;
                                          ^
```

**Comment by poettering** [comment]
rebased, applied @bluca's change, and fixed the compiler thing.

**Review by bluca — APPROVED** [review]
LGTM

**Pull request merged**